src/lib/constants.ts

```diff
@@ -83,7 +83,7 @@ export const stakingContracts = [
     owner: 'ShapeShift',
     contractAddress: '0xdd80e21669a664bce83e3ad9a0d74f8dad5d9e72',
     pool: poolsContractData['0x470e8de2ebaef52014a47cb5e6af86884947f08c'],
-    periodFinish: 1631329219,
```

Comment:
no need to hold this fix up, but is periodFinish something we can read from the contract or otherwise calculate?

Reply:
yeah it is on the contract but requires the roundtrip so we added it here to avoid the extra network request